### PR TITLE
Ocius and norship press release

### PR DESCRIPTION
**Pull request by r007**

### CODE REVIEW

**Comment by r007** [comment]
@cjmoses please approve the PR.

**Comment by cjmoses** [comment]
approve

File: src/blog/ocius-and-norship-press-release/index.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ featuredImage: image1.jpg
 
 Both companies are excited by this new collaboration which is designed to provide the assurance and confidence to end customers that Bluebottle USV’s can sustainably and reliably be deployed at sea for indefinite periods of time.
 
-> Robert Dane, OCIUS Chief Executive Officer said “Designing vessels and support systems with reliability, availability and maintainability in mind is paramount to the long term adoption and success of our USVs. The collaboration with Norship represents a further milestone in the “Bluebottle” development cycle. Our design and technological expertise will be further enhanced by Norships’ vast experience in maritime sustainment.”
```
Comment (cjmoses) [inline src/blog/ocius-and-norship-press-release/index.md:14]: remove the quote block

File: src/blog/ocius-and-norship-press-release/index.md [hunk 2]
```diff
@@ -11,6 +11,8 @@ featuredImage: image1.jpg
...
+* Robert Dane, OCIUS Chief Executive Officer said “Designing vessels and support systems with reliability, availability and maintainability in mind is paramount to the long term adoption and success of our USVs. The collaboration with Norship represents a further milestone in the “Bluebottle” development cycle. Our design and technological expertise will be further enhanced by Norships’ vast experience in maritime sustainment. *
 
-> Olav Groot, Norship Chief Executive Officer added: “autonomous, self-sustaining USVs which can carry a six-hundred-pound sensor payload present a number of unique opportunities and challenges from a sustainment and reliability perspective. With our engineering teams working closely together we are confident that customer expectations can be both met and exceeded.”
```
Comment (cjmoses) [inline src/blog/ocius-and-norship-press-release/index.md:16]: remove the quote block

File: src/blog/ocius-and-norship-press-release/index.md [hunk 3]
```diff
@@ -11,6 +11,8 @@ featuredImage: image1.jpg
 
 Both companies are excited by this new collaboration which is designed to provide the assurance and confidence to end customers that Bluebottle USV’s can sustainably and reliably be deployed at sea for indefinite periods of time.
 
-> Robert Dane, OCIUS Chief Executive Officer said “Designing vessels and support systems with reliability, availability and maintainability in mind is paramount to the long term adoption and success of our USVs. The collaboration with Norship represents a further milestone in the “Bluebottle” development cycle. Our design and technological expertise will be further enhanced by Norships’ vast experience in maritime sustainment.”
+* Robert Dane, OCIUS Chief Executive Officer said “Designing vessels and support systems with reliability, availability and maintainability in mind is paramount to the long term adoption and success of our USVs. The collaboration with Norship represents a further milestone in the “Bluebottle” development cycle. Our design and technological expertise will be further enhanced by Norships’ vast experience in maritime sustainment. *
```
Comment (cjmoses) [inline src/blog/ocius-and-norship-press-release/index.md:14]: changed to add italics

File: src/blog/ocius-and-norship-press-release/index.md [hunk 4]
```diff
@@ -11,6 +11,8 @@ featuredImage: image1.jpg
...
 
-> Olav Groot, Norship Chief Executive Officer added: “autonomous, self-sustaining USVs which can carry a six-hundred-pound sensor payload present a number of unique opportunities and challenges from a sustainment and reliability perspective. With our engineering teams working closely together we are confident that customer expectations can be both met and exceeded.”
+* Olav Groot, Norship Chief Executive Officer added: “autonomous, self-sustaining USVs which can carry a six-hundred-pound sensor payload present a number of unique opportunities and challenges from a sustainment and reliability perspective. With our engineering teams working closely together we are confident that customer expectations can be both met and exceeded.” *
```
Comment (cjmoses) [inline src/blog/ocius-and-norship-press-release/index.md:16]: changed to add italics

File: src/blog/ocius-and-norship-press-release/index.md [hunk 5]
```diff
@@ -11,6 +11,8 @@ featuredImage: image1.jpg
...
+
+![Bob and Bruce play follow the leader](image2.jpg)
```
Comment (cjmoses) [inline src/blog/ocius-and-norship-press-release/index.md:18]: added new image

**Review by cjmoses — APPROVED** [review]

**Pull request merged**